### ISSUE TITLE
Live stream: drift between MPD timeline and tfdt..

### ISSUE THREAD

**Issue by DanielMartine** [gpac/gpac#965]
Hello, I'm using MP4Box inside a web server to generate livestream.
I'm having trouble with the timeline after an hour since it started

After ffmpeg specific encoding i use:
`MP4Box -dash-live 4000 -rap -frag-rap -profile dashavc264:live -subdur 4000 -bs-switching no -time-shift 20 -out manifest.mpd video.mp4#video video.mp4#audio`

All work fine on all common browsers, except a warning about the process is sometime too late, but it don't seems to create any issue to dash.js that work fine:

```
Next generation scheduled in 3964 ms (DASH time 664215 ms)
Slept for 3964 ms before generation
[DASH] Generating segments and MPD 0.215235 seconds too late
[DASH] Removing segment out_dash_track1_160.m4s - 0.000234781 sec too late
[DASH] Removing segment out_dash_track1_161.m4s - -3.99977 sec too late
[DASH] Removing segment out_dash_track2_161.m4s - -2.02477 sec too late
DASH-ing files: 4.00s segments 4.00s fragments single sidx per segment
Spliting segments and fragments at GOP boundaries
DASHing file out.mp4
DASHing file out.mp4                                        
[DASH] Generating MPD at time 2018-01-16T12:43:25.215Z      
[DASH] Current Period Duration: 123.243
Next generation scheduled in 3974 ms (DASH time 668215 ms)
Slept for 3974 ms before generation
```

After one hour the fluidity of the playback degrades and terminal error is:

```
[DASH] Generating MPD at time 2018-01-16T13:37:53.535Z      
[DASH] Current Period Duration: 123.243
Next generation scheduled in 3973 ms (DASH time 3936533 ms)
Slept for 3973 ms before generation
[DASH] Removing segment out_dash_track2_981.m4s - -0.709 sec too late
DASH-ing files: 4.00s segments 4.00s fragments single sidx per segment
Spliting segments and fragments at GOP boundaries
DASHing file out.mp4
DASHing file out.mp4                                        
[DASH] Segment out_dash_track2_985.m4s (Number #985): drift between MPD timeline and tfdt exceeds 50% of segment duraion (MPD time minus TFDT 10.464333 secs) - bitstream will not be compliant, try using segment template or reencode
```

Do you now how can i make the streaming unstoppable? 
I run my tests on a macbook 13 of 2014 so I fear it may be the fault of the hardware, could confirm? 

Thanks,
Dan

**Comment by rbouqueau** [comment]
Tahks for reporting. Duplicate of #949

**Comment by DanielMartine** [comment]
Hi @rbouqueau i was reading #949 i fix my issue with -subdur double then -dash-live time:
MP4Box -dash-live 4000 -subdur 8000 -profile dashavc264:live -mpd-refresh 30.0 -bs-switching no -time-shift 80 -min-buffer 4000 -insert-utc -out manifest.mpd out.mp4#video out.mp4#audio
May i ask which is a good time for mpd-update? and if this command line is correct for live stream?
I'have some trouble to get a live skin for the player and i actually investigate every direction.
basically i need to disable seekbar and play pause, cause a pause too long could move the playhead outside the buffer duration. 
Any advise?
Thanks,
Dan
